README.md

```diff
@@ -11,3 +11,10 @@ flake8-f-string is a flake8 plugin to warn passing callable objects as arguments
 
 - Python 3
 - flake8 >= 3.5
+
+Error codes
+-----------
+
+This plugin is using the following error codes:
+
+- ERG100 - passing callable objects as arguments for formatted string
```

Comment:
Propose to not expose company name into error codes.

We are adding "f-string" plugin. So maybe `FS101` would be better?

Reply:
changed